packages/office-ui-fabric-react/src/components/MessageBar/MessageBar.styles.ts

```diff
@@ -84,21 +85,31 @@ export const getStyles = (props: IMessageBarStyleProps): IMessageBarStyles => {
     }
   };
 
-  const dismissalAndExpandStyle: IStyle = {
-    flexShrink: 0,
-    width: 32,
-    height: 32,
-    padding: '8px 12px',
-    selectors: {
-      '& .ms-Button-icon': dismissalAndExpandIconStyle,
-      ':hover': {
-        backgroundColor: 'transparent'
+  const dismissalAndExpandStyle: IStyle = [
+    getFocusStyle(theme, {
+      inset: 1,
+      highContrastStyle: {
+        outlineOffset: '-2px',
+        outlineColor: 'Highlight'
       },
-      ':active': {
-        backgroundColor: 'transparent'
+      borderColor: 'transparent'
+    }),
+    {
+      flexShrink: 0,
+      width: 32,
+      height: 32,
+      padding: '8px 12px',
+      selectors: {
+        '& .ms-Button-icon': dismissalAndExpandIconStyle,
+        ':hover': {
```

Comment:
I know this part is just moved but this selector could be combined with the next (`:hover, :active`)

Reply:
Combining them however has the adverse effect of making it harder for concatStyleSets to kick in, therefore making it harder to customize/overwrite these selectorts

Reply:
Oh interesting...mergeStyles is so hard...